source/ProgArgs.cpp

```diff
@@ -513,6 +513,10 @@ void ProgArgs::defineAllowedArgs()
 /*s*/	(ARG_FILESIZE_LONG "," ARG_FILESIZE_SHORT, bpo::value(&this->fileSizeOrigStr),
 			"File size. (Default: 0; supports base2 suffixes, e.g. \"2M\")")
 #ifdef S3_SUPPORT
+/*s3c*/	(ARG_S3CREDFILE_LONG, bpo::value(&this->s3CredentialsFile),
```

Comment:
this section is ordered by user option strings (the "--somename" behind the ARG_... definition) to print the user "--help-all" output with option names in alphabetical order.